src/main/kotlin/io/github/gradlenexus/publishplugin/InitializeNexusStagingRepository.kt

```diff
@@ -52,14 +51,13 @@ constructor(
     }
 
     internal fun createStagingRepo(): URI {
-        return serverUrlToStagingRepoUrl.computeIfAbsent(repository.get().nexusUrl.get()) { serverUrl ->
-            val client = NexusClient(serverUrl, repository.get().username.orNull, repository.get().password.orNull, clientTimeout.orNull, connectTimeout.orNull)
-            val stagingProfileId = determineStagingProfileId(client)
-            logger.info("Creating staging repository for stagingProfileId '{}'", stagingProfileId)
-            val stagingRepositoryIdAsString = client.createStagingRepository(stagingProfileId)
-            stagingRepositoryId.invoke(stagingRepositoryIdAsString)
-            client.getStagingRepositoryUri(stagingRepositoryIdAsString)
-        }
+        val serverUrl = repository.get().nexusUrl.get()
+        val client = NexusClient(serverUrl, repository.get().username.orNull, repository.get().password.orNull, clientTimeout.orNull, connectTimeout.orNull)
+        val stagingProfileId = determineStagingProfileId(client)
+        logger.info("Creating staging repository for stagingProfileId '{}'", stagingProfileId)
```

Comment:
What do you think of adding repository name and URL to the message?
I guess staging profile alone has little value for the end-user.

```suggestion
        logger.info("Creating staging repository for {} at {}, stagingProfileId '{}'", repository.get().name, serverUrl, stagingProfileId)
```

Reply:
Makes sense! 👍